bullet/src/SDFFeatures.cc

```diff
@@ -147,8 +149,8 @@ Identity SDFFeatures::ConstructSdfLink(
   world->addRigidBody(body.get());
 
   // Generate an identity for it
-  const auto linkIdentity = this->AddLink({name, _modelID, pose, mass,
-    linkInertiaDiag, myMotionState, collisionShape, body});
+  const auto linkIdentity = this->AddLink({name, _modelID, pose, inertialPose,
```

Comment:
I think you have to modify `AddLink` in the `Base.hh` file to have inertialPose as well

Reply:
I did, it would not compile otherwise